tutorials/preprocessing/plot_35_artifact_correction_regression.py

```diff
@@ -5,20 +5,37 @@
 Repairing artifacts with regression
 ===================================
 
-This tutorial covers removal of artifacts using regression as in
-:footcite:`GrattonEtAl1983`.
+This tutorial covers removal of artifacts using regression as in Gratton et al.
+1983 :footcite:`GrattonEtAl1983`.
 
 .. contents:: Page contents
    :local:
    :depth: 2
 
+Generally speaking, artifacts that result in time waveforms on the sensors
```

Comment:
...~~time~~ waveforms...

Reply:
I think it's helpful to make it as clear as possible we're talking about the time-domain signal here

Reply:
OK